src/main/java/org/example/domain/Book.java

```diff
@@ -0,0 +1,59 @@
+package org.example.domain;
+
+public class Book {
+    private int id;
+    private String title;
+    private String author;
+    private int page;
+    private BookState state;
+
+    public Book(int id, String title, String author, int page, BookState state) {
+        this.id = id;
+        this.title = title;
+        this.author = author;
+        this.page = page;
+        this.state = state;
+    }
+
+    @Override
+    public String toString() {
+        return this.id+","+this.title+","+this.author+","+this.page +","+this.state;
+    }
+
+    public String printBook() {
+        return "도서번호 : " + this.getId() + "\n제목 : " + this.getTitle()+"\n작가 이름 : " + this.getAuthor()
+        + "\n페이지 수 : " + this.getPage() +" 페이지" + "\n상태 : " + this.getState() + "\n------------------------------\n";
+    }
+
+    public boolean isOrganizing() {
+        if(this.state == BookState.ORGANIZING) return true;
+        else return false;
+    }
```

Comment:
```suggestion
    public boolean isOrganizing() {
         return this.state == BookState.ORGANIZING;
    }
```
이렇게 쓰는게 더 깔끔하지 않을까요?